internal/bminventory/inventory.go

```diff
@@ -3270,6 +3284,30 @@ func (b *bareMetalInventory) updateHostsData(ctx context.Context, params install
 	return nil
 }
 
+func (b *bareMetalInventory) updateClusterNetworkVMUsage(cluster *common.Cluster, usages map[string]models.Usage) error {
+	vmHosts := make([]string, 0)
+	userManagedNetwork := cluster.UserManagedNetworking != nil && *cluster.UserManagedNetworking
+	if userManagedNetwork {
+		for _, host := range cluster.Hosts {
+			hostInventory, err := common.UnmarshalInventory(host.Inventory)
+			if err != nil {
```

Comment:
Have you checked that when you call this at initialization time you have an inventory?

Reply:
When there is no inventory - it will cause an error - that will be logged. And continue.